events/elf_destiny_main_story_events.txt

```diff
@@ -274,6 +271,12 @@ elf_destiny_main_story_events.0021 = { # Restore the Portal part 2
 		save_scope_as = scoped_ruler
 	}
 
+	option = { # Receive Blessing Choice
+		name = elf_destiny_main_story_events.0021.ok_move_portal
+		set_realm_capital = title:c_martyuba
```

Comment:
I think this is a neat idea but this option would need a trigger that checks if you haven't already moved the portal to another county.

So it would check if the portal is in  title:c_martyuba